.meta.toml

```diff
@@ -0,0 +1,21 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
+[meta]
+template = "pure-python"
+commit-id = "519ffffe1b360ede58b32a375b5f8b26059d8016"
+
+[python]
+with-appveyor = false
+with-pypy = true
+with-legacy-python = true
+with-docs = false
+with-sphinx-doctests = false
+
+[coverage]
+fail-under = 87
+
+[github-actions]
+additional-install = [
+    "sudo apt-get update",
+    "sudo apt-get install libxml2-dev libxslt-dev",
```

Comment:
Have these dependencies been installed on travis by default?

Reply:
I don't know but possibly they have been there.
They are needed to build lxml for PyPy.